js/src/components/app-button/index.js

```diff
@@ -49,9 +49,14 @@ const AppButton = ( props ) => {
 		onClick( ...args );
 	};
 
+	const classes = [ 'app-button', className ];
+	if ( rest.iconPosition === 'right' ) {
+		classes.push( 'gla-app-button--fix-icon-position' );
```

Comment:
The `app-button` naming is too general for me and I thought we already had a consensus to have `gla-` prefix for our shared components. About the `fix-icon-position`, I think the **fix** still needs to point out it's a style fix. So I renamed it to `app-button--fix-icon-position-right` by https://github.com/woocommerce/google-listings-and-ads/pull/918/commits/051e2c7f2339b791136607a40ee83b508b17f52e. The `gla-` prefix could be discussed later if you don't think it's needed.

Reply:
> The `app-button` naming is too general for me and I thought we already had a consensus to have `gla-` prefix for our shared components.

The way I think of them is: 

- `app-` prefix: a generic prefix meant for a generic component, it can possibly be used in other projects (copy and paste), or might even be contributed upstream to WordPress or WooCommerce components library.
- `gla-` prefix: very specific to GLA, unlikely to be reused outside of GLA.

Yeah I agree with you that we can discuss this later.